template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/IdentityServiceConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Company.WebApplication1.Identity
+{
+    public class IdentityServiceConstants
```

Comment:
You might change the client ones so I am splitting those out. Even the service ones might not be fixed for all time, so I will make the static readonly.